texthero/visualization.py

```diff
@@ -177,9 +190,7 @@ def top_words(s: pd.Series, normalize=False) -> pd.Series:
 
     # TODO replace it with tokenizer.
 
-    pattern = (
-        rf"((\w)[{string.punctuation}](?:\B|$)|(?:^|\B)[{string.punctuation}](\w))"
-    )
+    pattern = rf"((\w)[{string.punctuation}](?:\B|$)|(?:^|\B)[{string.punctuation}](\w))"
```

Comment:
Should be black-formatted

Reply:
thanks @jbesomi, totally forgot that. Done formatting with black, please check last commit. thanks!